osu.ElasticIndexer/schemas/solo_scores.json

```diff
@@ -26,7 +26,7 @@
         "type": "date"
       },
       "id": {
-        "type": "long"
+        "type": "keyword"
```

Comment:
I think the id should still be long because it's used for the tie breaker.

Reply:
`keyword` works fine for sort and cursor

Reply:
of course this makes the assumption that no one tries to enter a string into the field and mess it up 🤔